### PR TITLE
ceph - point config at ceph server

### PR DESCRIPTION
**Pull request by bwalsh**
* also included updates to dvc files necessary after `dvc repro outputs.bmeg_manifest.dvc`
* small tweak to source.ensembl-protein.homo_sapiens.json.dvc added dependency to `source/ensembl-protein/version.txt`

### CODE REVIEW

File: README-DVC.md [hunk 1]
```diff
@@ -577,111 +596,6 @@ dvc run --file outputs.tcga.expression.dvc --yes \
   -o outputs/tcga/UVM.ExpressionOf.Edge.json.gz \
   "python3 transform/tcga/expression.py"
 #
-dvc run --file outputs.tcga.gistic2cna.dvc --yes \
```
Comment (adamstruck) [inline README-DVC.md]: Why is this block being removed?

File: outputs.bmeg_manifest.dvc [hunk 2]
```diff
@@ -308,140 +308,136 @@ deps:
...
-- md5: d99490356b48d72e62556fd7c3471cb7
+- md5: eb869fafc48d8476f91175ff5381d858
   path: outputs/tcga/TCGA-ACC.CopyNumberAlteration.Vertex.json.gz
-- md5: 1dc3ddcebd7185200844e8fc980a40c8
-  path: outputs/tcga/TCGA-ACC.CopyNumberAlterationOf.Edge.json
-- md5: 5bf0299eee8488abb12188abe419c5f8
+- md5: ce9cfd1818014fb1425856d90bb375a5
   path: outputs/tcga/TCGA-ACC.CopyNumberAlterationOf.Edge.json.gz
-- md5: b27e006e080d38763b179521d0b1ede9
+- md5: 41d0ae4615f2f0b537962846cfef4465
```
Comment (adamstruck) [inline outputs.bmeg_manifest.dvc]: Any idea why all of these md5 sums changed?

File: README-DVC.md (hunk 1 above)
Comment (bwalsh) [reply README-DVC.md]: I'll check. I generated the commands from the files...

Comment (bwalsh) [reply README-DVC.md]: Found the issue on how I was regenerating the provenance commands.

File: outputs.bmeg_manifest.dvc (hunk 2 above)
Comment (bwalsh) [reply outputs.bmeg_manifest.dvc]: I'm not sure.   I pulled from master, pulled from (dvc minio) and did a `dvc repro outputs.bmeg_manifest.dvc` and it indicated gistic2cna needed to be re-run.

File: README-DVC.md (hunk 1 above)
Comment (bwalsh) [reply README-DVC.md]: Added fix for #271

**Pull request merged**